discovery-provider/src/api/v1/tracks.py

```diff
@@ -160,6 +195,36 @@ def get(self, track_id):
         return get_single_track(decoded_id, current_user_id, full_ns)
 
 
+full_track_slug_parser = reqparse.RequestParser()
+full_track_slug_parser.add_argument("handle", required=True)
+full_track_slug_parser.add_argument("slug", required=True)
+
+
+@full_ns.route("/")
+class FullTrackBySlug(Resource):
+    @record_metrics
+    @full_ns.marshal_with(full_track_response)
+    @cache(ttl_sec=5)
+    def get(self):
+        args = full_track_slug_parser.parse_args()
+        slug, handle = args.get("slug"), args.get("handle")
+        if not (slug and handle):
+            full_ns.abort(400, "Missing required param slug or handle")
+        tracks = get_tracks(
```

Comment:
we probably want to pass along the current_user_id with these requests too so that fields like "has_current_user_reposted" get captured